torch/nn/modules/rnn.py

```diff
@@ -313,7 +313,7 @@ class RNN(RNNBase):
     function:
 
     .. math::
-        h_t = \tanh(W_{ih} x_t + b_{ih} + W_{hh} h_{(t-1)} + b_{hh})
+        h_t = \tanh((x_t)W_{ih}^T + b_{ih} + h_{(t-1)}W_{hh}^T + b_{hh})
```

Comment:
Might be too pedantic, but I think it reads slightly better without the extra parens:

```suggestion
        h_t = \tanh(x_t W_{ih}^T + b_{ih} + h_{t-1}W_{hh}^T + b_{hh})
```